### ISSUE TITLE
Bring back input field labels for a11y

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#6311]
@MarcoZehe informed me that [placeholders have a lot of a11y problems](https://www.smashingmagazine.com/2018/06/placeholder-attribute/) and that we should bring labels back in some form to help screen reader users.

**Comment by MarcoZehe** [comment]
And not just screen reader users, as is apparent from the article linked to above. Bigger touch (or click) targets help users with motor impairments as well, as well as aging people with less accuracy in hand-eye coordination. And yes, Mozillians get older, too. ;)

**Comment by vbudhram** [comment]
from mtg:

In the photon pull request, labels were removed, we should be able to bring them back, ref https://github.com/mozilla/fxa-content-server/pull/6175/

**Comment by championshuttler** [comment]
Hi @shane-tomlinson I would like to work on it, can i take this up?


Thanks

**Comment by lmorchard** [comment]
@championshuttler Hi there! I can help here too. Feel free to start working on this. I'm going to assign this issue to myself to help track it, since it looks like GitHub only lets us assign to core team members.

**Comment by DraciVik** [comment]
Hi @championshuttler and @lmorchard . I am an Outreachy applicant and I'm looking for bugs to solve so I can contribute. If you're too busy with other bugs or because you are on a vacation according to your GitHub profile @championshuttler maybe I can give this a try? Thank you!

**Comment by championshuttler** [comment]
Hi @DraciVik I am working on it!

Thanks
